docs/tasks/Base.md

```diff
@@ -56,7 +56,9 @@ $this->taskExecStack()
 * `stopOnFail($stopOnFail = null)`   * `param bool` $stopOnFail
 * `result($result)` 
 * `dir($dir)`  Changes working directory of command
-* `printed($arg)`  Should command output be printed
```

Comment:
If you want to rename a function, mark it deprecated, don't take it out of the docs entirely.